tests/verilog/block_labels.ys

```diff
@@ -0,0 +1,26 @@
+read_verilog <<EOT
+module foo;
+
+	genvar a = 0;
+	for (a = 0; a < 10; a++) begin : a
+	end : a
+endmodule
+EOT
+read_verilog <<EOT
+module foo2;
+
+	genvar a = 0;
+	for (a = 0; a < 10; a++) begin : a
+	end
+endmodule
+EOT
+
+logger -expect error "Begin label (.*) and end label (.*) don't match." 1
```

Comment:
Minor nit: Can the pattern match `a` and `b` specifically, rather than `.*`? Also note that the `.` at the end of this pattern is being treated as a wildcard, so perhaps either escape it or remove it.

Reply:
Done. I also escaped ``.`` at the end.